package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "app",
+  "private": "true",
+  "scripts": {
+    "build": "webpack --config webpack.config.js"
+  },
+  "dependencies": {
+    "@hotwired/stimulus": "^3.2.1",
+    "@hotwired/turbo-rails": "^7.3.0",
+    "@reduxjs/toolkit": "^1.9.3",
+    "axios": "^1.3.4",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-redux": "^8.0.5",
+    "webpack": "^5.76.3",
+    "webpack-cli": "^5.0.1"
```

Comment:
- [ Required ] I think it would be better if you added the `react-router-dom` library as a dependency in your `package.json` file, I experienced the error below when I ran `./bin/dev` to start the application. Please consider fixing this.

## Error
![Screenshot from 2023-03-30 00-53-09](https://user-images.githubusercontent.com/16392046/228677185-13256036-695c-4ef7-a873-13bdd066ab7b.png)

Reply:
This was a eye opener of something I was missing. Thank you for the feedback